server/dbutil/dbutil.go

```diff
@@ -6,23 +6,47 @@ import (
 	"fmt"
 	"log"
 	"os"
+	"regexp"
+	"strings"
 
+	_ "github.com/lib/pq"
```

Comment:
It looks like this PR adds a new dependency library.

As https://github.com/src-d/code-annotation/pull/33 is already merged, this means import/export tools should also use `dep` and thus adding a dependency would include modification of the .toml file, right?

Reply:
makes sense. I'll rebase from master and update dep.